codec-dns/src/main/java/io/netty/handler/codec/dns/DnsRecord.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2015 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.handler.codec.dns;
+
+/**
+ * Represents any resource record (answer, authority, or additional resource
+ * records).
+ */
+public interface DnsRecord {
+
+    /**
+     * DNS resource record class: {@code IN}
+     */
+    int CLASS_IN = 0x0001;
+
+    /**
+     * DNS resource record class: {@code CSNET}
+     */
+    int CLASS_CSNET = 0x0002;
+
+    /**
+     * DNS resource record class: {@code CHAOS}
+     */
+    int CLASS_CHAOS = 0x0003;
+
+    /**
+     * DNS resource record class: {@code HESIOD}
+     */
+    int CLASS_HESIOD = 0x0004;
+
+    /**
+     * DNS resource record class: {@code NONE}
+     */
+    int CLASS_NONE = 0x00fe;
+
+    /**
+     * DNS resource record class: {@code ANY}
+     */
+    int CLASS_ANY = 0x00ff;
```

Comment:
I guess adding `.` at the end of the field description in this case will be unnatural, because of `:`